src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1497,6 +1497,20 @@ def use_mutator_plugin(target_name, extra_env):
   return True
 
 
+def use_radamsa_in_process(extra_env):
+  """Decide whether to use Radamsa in process. If yes, add the path to the
+  radamsa so to LD_PRELOAD in |extra_env| and return True."""
+
+  #Like above. Not sure if there is a reason why this would work for windows
```

Comment:
Spaces after `#`. I'm very surprised linting/formatting didn't catch this.
I'm not sure what `Like above` means in this context. And just say that `mutator plugin doesn't work on windows` and leave out the `Not sure if there is a reason why this would...` part, doesn't add anything and it seems sarcastic :-)
I think you are actually asking a question though, so the answer is that it wont work on Windows.

Reply:
If you have a question for reviewers it's probably better to do so using the code review tool rather than leaving it in source. It's more likely to be seen this way and can't be accidentally left in the code.